orchprovider/k8s/v1/k8s.go

```diff
@@ -800,6 +800,8 @@ func (k *k8sOrchestrator) createControllerDeployment(volProProfile volProfile.Vo
 		return nil, err
 	}
 
+	pvc := vol.Labels.K8sPersistentVolumeClaim
```

Comment:
Can we get the pvc name from the volProProfile structure?

Reply:
we are using volume struct which is returned by the volume() method of volProProfile interface.
This volume struct holds the user's request for a OpenEBS volume.